src/mmw/apps/geoprocessing_api/schemas.py

```diff
@@ -178,3 +178,61 @@
     },
     required=['location'],
 )
+
+nlcd_override_allowed_values = '", "'.join([
```

Comment:
This is currently only used for generating the documentation. This should also be used for validating the input.